pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -1883,6 +1962,48 @@ jobs:
         RABBITMQ_CONFIG_VALUES_FILE: rabbitmq/values.yml
       input_mapping: {census-rm-kubernetes-dependencies-repo: census-rm-kubernetes-dependencies-repo}
 
+
+  # Apply Database Patches
+- name: "CI Apply Database Patches"
+  serial: true
+  serial_groups: [ci-database-patches,
+                  ci-acceptance-tests,
+                  action-scheduler-deploy,
+                  action-worker-deploy,
+                  case-api-deploy,
+                  case-processor-deploy,
+                  uac-qid-service-deploy,
+                  pubsub-deploy,
+                  print-file-service-deploy,
+                  fieldwork-adapter-deploy,
+                  notify-processor-deploy,
+                  notify-stub-deploy,
+                  exception-manager-deploy,
+                  toolbox-deploy,
+                  database-monitor-deploy,
+                  rabbitmonitor-deploy,
+                  regional-counts-deploy]
+  on_failure: *slack_failure_alert_ci
+  on_error: *slack_error_alert_ci
+  plan:
+  - get: census-rm-ddl-master
+    trigger: true
+    passed: ["Build DDL Latest"]
+  - get: census-rm-kubernetes-repo
```

Comment:
Could use existing census-rm-kubernetes-repo instead

Reply:
Changed to `census-rm-kubernetes-optional-repo` instead